src/Utilities/HGeoUtil.f90

```diff
@@ -0,0 +1,110 @@
+!> @brief General-purpose hydrogeologic functions.
+module HGeoUtilModule
+  use KindModule, only: DP, I4B
+  use ConstantsModule, only: DZERO, DONE
+
+  implicit none
+  private
+  public :: hyeff
+
+contains
+
+  !> @brief Calculate the effective horizontal hydraulic conductivity from an
+  !! ellipse using a specified direction (unit vector vg1, vg2, vg3)
+  !!
+  !! k11 is the hydraulic conductivity of the major ellipse axis
```

Comment:
We should talk to @aprovost-usgs and come up with more general names for this routine and the argument variables.  These are generic ellipsoid properties that apply to hydraulic conductivity or solute dispersion or energy dispersion.  Good idea to move the routine out of NPF as it doesn't belong there any more.  We can do the renaming later in order to keep this moving.

Reply:
Good idea, I'll think about that